package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "yes",
+  "version": "1.0.0",
+  "description": "SECURACCOUNT (securaccount.com)",
+  "main": "index.js",
```

Comment:
Probably you already did, but don't forget to point to server.js instead of index.js